### PR TITLE
AMBARI-26020: upgrade Ambari Metrics to python3

### PR DESCRIPTION
**Pull request by JiaLiangC**
## What changes were proposed in this pull request?
upgrade Ambari Metrics to python3
(Please fill in changes proposed in this fix)

## How was this patch tested?
manual test
image
image
image

(Please explain how this patch was tested. Ex: unit tests, manual tests)
(If this patch involves UI changes, please attach a screen-shot; otherwise, remove this)

Please review [Ambari Contributing Guide](https://cwiki.apache.org/confluence/display/AMBARI/How+to+Contribute) before opening a pull request.

### CODE REVIEW

File: ambari-metrics-host-monitoring/src/main/python/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```
[the remark this answers is not in the document]
Comment (JiaLiangC) [reply ambari-metrics-host-monitoring/src/main/python/__init__.py:1]: This is done to maintain consistency with the previous code. The prior code, including the upgraded Ambari server, all use `#!/usr/bin/env python`. Therefore, it has been changed to `#!/usr/bin/env python3`.

**Comment by JiaLiangC** [comment]
@virajjasani Ambari python3 feature already merged, so could you help review this pr?

**Review by virajjasani — APPROVED** [review]
+1, skimmed through the changes. Let's wait for a day before merging it.

**Comment by wenpeiHuang** [comment]
我编译了26020这个版本，我的ambari为3.0.0，bigtop为3.3.0， 但是当我启动ambari-metrics 的时候报错
NoneType: None

The above exception was the cause of the following exception:

Traceback (most recent call last):
  File "/var/lib/ambari-agent/cache/common-services/AMBARI_METRICS/3.0.0/package/scripts/metrics_collector.py", line 90, in <module>
    AmsCollector().execute()
  File "/usr/lib/ambari-agent/lib/resource_management/libraries/script/script.py", line 357, in execute
    self.execute_prefix_function(self.command_name, 'post', env)
  File "/usr/lib/ambari-agent/lib/resource_management/libraries/script/script.py", line 388, in execute_prefix_function
    method(env)
  File "/usr/lib/ambari-agent/lib/resource_management/libraries/script/script.py", line 430, in post_start
    raise Fail("Pid file {0} doesn't exist after starting of the component.".format(pid_file))
resource_management.core.exceptions.Fail: Pid file /var/run/ambari-metrics-collector/hbase-ams-regionserver.pid doesn't exist after starting of the component.

**Pull request merged**